view/theme/frio/css/style.css

```diff
@@ -70,6 +70,11 @@ video {
 	max-width: 100%;
 	max-height: 85vh;
 }
+
+.vjs-tech {
+	max-height: 100%;
+}
```

Comment:
Thought: Instead of adding it specifically to frio and vier, how about putting it in global.css?